cli/lidoStatus_test.go

```diff
@@ -0,0 +1,148 @@
+/*
+Copyright 2022 Nethermind
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cli
+
+import (
+	"bytes"
+	"io"
+	"math/big"
+	"testing"
+
+	"github.com/shopspring/decimal"
+	log "github.com/sirupsen/logrus"
+)
+
+type flags struct {
+	rewardAddress    string
+	networkName      string
+	longDescriptions bool
+}
+
+type lidoStatusCmdTestCase struct {
+	name  string
+	flags flags
+	isErr bool
+}
+
+func (f flags) argsList() []string {
+	s := make([]string, 0)
+	s = append(s, f.rewardAddress)
+	if f.networkName != "" {
+		s = append(s, "--network", f.networkName)
+	}
+	if f.longDescriptions {
+		s = append(s, "--l")
+	}
+
+	return s
+}
+
+func TestLidoStatusCmd(t *testing.T) {
+	tcs := []lidoStatusCmdTestCase{
+		{
+			name: "valid lido-status flags with long descriptions, Holesky",
+			flags: flags{
+				rewardAddress:    "0xC870Fd7316956C1582A2c8Fd2c42552cCEC70C88",
+				networkName:      "holesky",
+				longDescriptions: true,
+			},
+			isErr: false,
+		},
+		{
+			name: "valid lido-status flags, Holesky",
+			flags: flags{
+				rewardAddress: "0x22bA5CaFB5E26E6Fe51f330294209034013A5A4c",
+				networkName:   "holesky",
+			},
+			isErr: false,
+		},
+		{
+			name: "Invalid: missing address, Holesky",
+			flags: flags{
+				networkName: "holesky",
+			},
+			isErr: true,
+		},
+		{
+			name: "Invalid: zero address, Holesky",
+			flags: flags{
+				rewardAddress: "0x0000000000000000000000000000000000000000",
+				networkName:   "holesky",
+			},
+			isErr: true,
+		},
+	}
+
+	for _, tc := range tcs {
+		t.Run(tc.name, func(t *testing.T) {
+			log.SetOutput(io.Discard)
+			logsOut := new(bytes.Buffer)
+			tableOut := new(bytes.Buffer)
+			rootCmd := RootCmd()
+			rootCmd.SetOut(tableOut)
+			rootCmd.AddCommand(LidoStatusCmd())
+			argsL := append([]string{"lido-status"}, tc.flags.argsList()...)
+			rootCmd.SetArgs(argsL)
+			initLogging()
+			log.SetOutput(logsOut)
+			rootCmd.SetOutput(io.Discard)
+			err := rootCmd.Execute()
+			if tc.isErr && err == nil {
+				t.Error("sedge lido-status expected to fail")
+			} else if !tc.isErr && err != nil {
+				t.Errorf("sedge lido-status failed: %v", err)
+			}
+		})
+	}
+}
+
+func TestWeiToEth(t *testing.T) {
```

Comment:
Please add test cases for bigger values, like 1.5 ETH, 1 ETH. 200 ETH

Reply:
[Done](https://github.com/NethermindEth/sedge/pull/402/commits/7880b17ce9c0c5689a9bf5f6e92a300813f2b95e)